metals/src/main/scala/scala/meta/internal/bsp/BspServers.scala

```diff
@@ -65,23 +66,30 @@ final class BspServers(
   ): Future[BuildServerConnection] = {
 
     def newConnection(): Future[SocketConnection] = {
-      val process = new ProcessBuilder(details.getArgv)
-        .directory(projectDirectory.toFile)
-        .start()
+      scribe.info(s"Running BSP server ${details.getArgv}")
+
+      val proc = SystemProcess.run(
+        details.getArgv.asScala.toList,
+        projectDirectory,
+        redirectErrorOutput = false,
+        Map(),
+        processOut = None,
+        processErr = Some(l => scribe.info("BSP server: " + l)),
```

Comment:
Shouldn't it be `scribe.error`?

Reply:
I'm not sure. It's stderr, because it's the only place a BSP server can print in the console, but it doesn't mean there's an actual error here. It's whatever the BSP server wants to print.